internal/plan/fallback.go

```diff
@@ -8,10 +8,10 @@ import (
 
 // CreateFallbackPlan creates a fallback test plan for the given tests and parallelism.
 // It distributes test cases evenly accross the tasks using deterministic algorithm.
-func CreateFallbackPlan(testCases []TestCase, parallelism int) TestPlan {
+func CreateFallbackPlan(files []string, parallelism int) TestPlan {
```

Comment:
We don't need an array of a whole test struct to create a fallback plan, so I refactored it to takes an array of file name instead.